assets/xml/objects/object_hakugin_demo.xml

```diff
@@ -1,8 +1,8 @@
 ﻿<Root>
     <File Name="object_hakugin_demo" Segment="6">
-        <Animation Name="object_hakugin_demo_Anim_001420" Offset="0x1420" />
-        <Animation Name="object_hakugin_demo_Anim_001A4C" Offset="0x1A4C" />
-        <Animation Name="object_hakugin_demo_Anim_002704" Offset="0x2704" />
-        <Animation Name="object_hakugin_demo_Anim_003378" Offset="0x3378" />
+        <Animation Name="gHakuginShowAnim" Offset="0x1420" />
```

Comment:
If we can decide on a reasonable term to prefix things with, we tend to avoid using the object name in symbols like this. For example, we call it `gGoronShiverAnim` and not `gObjectOF1dMapShiverAnim`. With that being said, can you think of a good prefix for these animations? They seem to be animations for the cutscene when Spring comes to the mountains. Maybe `gSpringCutsceneGoron`? But I'll let you think on it and make the final call.